persist/boltdb_test.go

```diff
@@ -0,0 +1,167 @@
+package persist
+
+import (
+	"math/rand"
+	"os"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/bolt"
+)
+
+const numTestInputs = 25
+
+var dbVersions = []string{"0.0.0", "7.0.4", "asdf"}
+
+type testInput struct {
+	dbMetadata Metadata
+	dbFilename string
+}
+
+var testInputs = make([]testInput, numTestInputs)
+
+var testBuckets = [][]byte{
+	[]byte("FakeBucket"),
+	[]byte("FakeBucket123"),
+	[]byte("FakeBucket123!@#$"),
+	[]byte("Another Fake Bucket"),
+	[]byte("FakeBucket" + RandomSuffix()),
+	[]byte("_"),
+	[]byte(" asdf"),
+}
```

Comment:
Since these variables and costs are only used inside of one test, it's better to declare them inside that test, especially since they don't really have broader application.